lua/lsp/init.lua

```diff
@@ -111,7 +111,10 @@ autocmd BufWritePre *.lua lua vim.lsp.buf.formatting_sync(nil, 100) ]]
 -- Java
 -- autocmd FileType java nnoremap ca <Cmd>lua require('jdtls').code_action()<CR>
 
-local function documentHighlight(client, bufnr)
+local function get_lsp_document_highlighting(client, bufnr)
```

Comment:
How about `lsp_highlight_document` or `highlight_document` ?
My problem with this one is that it always returns nil, so it doesn't really get something.